src/sbvr-api/sbvr-utils.ts

```diff
@@ -1352,15 +1442,15 @@ export const handleODataRequest: Express.Handler = async (req, res, next) => {
 
 			// Otherwise its a multipart request and we reply with the appropriate multipart response
 		} else {
-			(res.status(200) as any).sendMulti(
-				responses.map((response) => {
+			res.status(200).json({
+				responses: responses.map((response) => {
 					if (response instanceof HttpError) {
-						response = httpErrorToResponse(response);
+						return httpErrorToResponse(response);
```

Comment:
This can be merged in a separate PR

Reply:
See: https://github.com/balena-io/pinejs/pull/679